django-stubs/contrib/sessions/base_session.pyi

```diff
@@ -11,9 +11,9 @@ class BaseSessionManager(models.Manager[_T]):
     def save(self, session_key: str, session_dict: dict[str, Any], expire_date: datetime) -> _T: ...
 
 class AbstractBaseSession(models.Model):
-    expire_date: datetime
-    session_data: str
-    session_key: str
+    session_key = models.CharField(primary_key=True)
+    session_data = models.TextField()
+    expire_date = models.DateTimeField()
     objects: Any
```

Comment:
Also let's remove the `objects` member, it's defined in the `Model` class.

```suggestion
```

Reply:
It shouldn't be removed, I think, but rather set to:

```suggestion
    objects: ClassVar[BaseSessionManager[Self]]
```